VideoLocker/res/layout/registration_spinner_item.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    style="?android:attr/spinnerItemStyle"
+    tools:targetApi="17"
+    android:singleLine="true"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:ellipsize="marquee"
+    android:textAlignment="inherit"
+    android:fontFamily="sans-serif"
```

Comment:
Which part are you referring to? The main reason I added in this XML was for android:fontFamily="sans-serif" (i.e. Roboto Regular). The other attributes makes sure that we don't lose anything when switching from the default item view to this one.